connector/src/main/scala/com/microsoft/kusto/spark/utils/ExtendedKustoClient.scala

```diff
@@ -205,6 +205,8 @@ class ExtendedKustoClient(val engineKcsb: ConnectionStringBuilder, val ingestKcs
     var delayPeriodBetweenCalls = DelayPeriodBetweenCalls
     var consecutiveSuccesses = 0
     val useMaterializedViewFlag = shouldUseMaterializedViewFlag(database, targetTable, crp)
+    var firstMoveRetries = writeOptions.maxRetriesOnMoveExtents
```

Comment:
Nits, this and the one below can be vals. 
Yifat wanted the second threshold to be 10